hack/sigstore-setup/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is copied from sigstore/sigstore
+# See https://github.com/sigstore/sigstore/blob/main/test/e2e/docker-compose.yml
+
+version: "3.8"
+services:
+  vault:
+    image: quay.io/sallyom/vault:latest
+    environment:
+      VAULT_DEV_ROOT_TOKEN_ID: ${VAULT_TOKEN}
+    ports:
+      - 8200:8200
+    privileged: true
+  localstack:
+    image: quay.io/sallyom/localstack:0.12.16
+    ports:
+      - 4566:4566
+    environment:
+      - SERVICES=kms
+  dex:
+    image: quay.io/sallyom/dexidp-dex:v2.31.0
+    ports:
+      - "5556:5556"
+    volumes:
+      - ./dex-config.yml:/etc/dex/dex-config.yml:z
```

Comment:
Do we need this `dex-config.yml`?

Reply:
nope, I've removed this and also updated the gist test script (not included in this PR but to prove this setup works until skopeo adds sigstore e2es) to only include the sigstore non-interactive e2es